### PR TITLE
Fix order of unescaping bytes

### PR DESCRIPTION
**Pull request by aawdls**
Ref #59

### CODE REVIEW

File: test/test_pb.py [hunk 1]
```diff
@@ -69,6 +69,16 @@ def test_escape_bytes_handles_example_unescaped_bytes():
     assert pb.escape_bytes(test_bytes) == expected
 
 
+def test_unescape_bytes_works_in_correct_oder_and_is_reversible():
+    test_bytes = b'hello \x1B\x01\x1B\x02 bye \x1B\x01\x1B\x03'
+    expected = b'hello \x1B\x0A bye \x1B\x0D'
+    unescpaed = pb.unescape_bytes(test_bytes)
+    assert unescpaed == expected
```
Comment (willrogers) [inline test/test_pb.py]: unescpaed

File: aa/pb.py [hunk 2]
```diff
@@ -62,13 +62,21 @@
 NL_BYTE = b'\x0A'
 CR_BYTE = b'\x0D'
 
-# The characters sequences required to unescape AA pb file format.
-PB_REPLACEMENTS = collections.OrderedDict([
+# The characters sequences required to unescape & escape AA pb file format.
```
Comment (willrogers) [inline aa/pb.py]: characters sequences

**Comment by willrogers** [comment]
Looks good, just amend the commit with the typos.

**Comment by aawdls** [comment]
In addition to the typos I also had left out an update to the test case which better demonstrates the problem. I've included that now.

**Pull request merged**